getting-started/sandbox-quick-start-guide.md

```diff
@@ -0,0 +1,439 @@
+# INTRODUCTION
+
+**Cloud File Transfer (CFT)** [sandbox][glossary.md] is a test environment that is almost identical to our production environment except that it supports only internet to internet transfers.
+
+In the following sections you will learn how to use APIs in the sandbox to send and receive files. \(Includes resource information on each of the API endpoints, request headers, response schema, sample request and response payloads!\)
+
+Try out our APIs and start transferring files in minutes!
+
+
+If you haven't yet signed up, fill up the [onboarding form.](https://form.gov.sg/#!/60a4cca76179d60012cdacac/preview)
+
+We will send you an email with your API credentials and account information.
+
+
+# Getting Started
+
+## Sandbox Credentials
+Refer to your welcome email for the following information:
+
+- [API Key][glossary.md]
+- [Client Id and Secret][glossary.md]
+- [API Gateway Id][glossary.md]
+- Sandbox (Internet).postman_environment.json*
+- CFT API v1.postman_collection.json*
+* Requires Postman application to be installed.
+
+## Setup Test Environment
+We recommend Postman for its user-friendly, simple interface. It supports most HTTP methods and several status codes for you to verify your response.
+
+### 1. Install Postman Application
+Download and install [Postman.][www.postman.com]
+
+### 2. Import CFT API v1 Collection and Sandbox Environment
+- Click **Collections -> Import -> Folder -> \"CFT API v1.postman_collection.json\"**
+- Click **Environment -> Import -> Folder -> \"Sandbox (Internet).postman_environment.json\"**
+Refer Sandbox Credentials.
```

Comment:
is this line correct and necessary?

this line appears differently in https://github.com/GovTechSG/CFT-Docs/blob/Quick-Start-Guide/getting-started/sandbox-quick-start-guide.md

its displayed as "*See Sandbox Account Information"